test/Constraints/optional.swift

```diff
@@ -565,3 +565,18 @@ func testFunctionContainerMethodCall(container: FunctionContainer?) {
   // expected-note@-3 {{coalesce}}
   // expected-note@-4 {{force-unwrap}}
 }
+
+// Test for Issue: 60730 (https://github.com/apple/swift/issues/60730)
+// Rdar: rdar://94037733
+func issue60730Test() {
+  struct S: P {}
+  func takesP(_: any P) {}
+  func passOptional(value: (any P)?) {
+    takesP(value)
+    // Add full error message
+    // Don't include the error message yet - run it and see the test fail first, and go from there.
```

Comment:
Is this outdated?

Reply:
Sorry! So embarrassing leaving the comments in there! Late night PR's :/

Reply:
No need to apologize, just making sure it's ready for review is all.